tests/functional/spec/features/calculated_summary_test_case.js

```diff
@@ -346,7 +346,13 @@ class TestCase {
       await browser.url(CalculatedSummaryTotalConfirmation.url());
       await expect(await browser.getUrl()).to.contain(CalculatedSummaryTotalConfirmation.pageName);
       const content = await $("h1 + ul").getText();
-      const textsToAssert = ["Total currency values: £25.92", "Total unit values: 1,467", "Total percentage values: 79", "Total number values: 124.58"];
+      const textsToAssert = [
```

Comment:
Minor and not related to this change - but would be good to add the same for format_percentage and assert it here so that it is consistent with the others?

Reply:
Added that now, thanks.